builtin/logical/database/backend.go

```diff
@@ -127,6 +131,110 @@ func Backend(conf *logical.BackendConfig) *databaseBackend {
 		WALRollback:       b.walRollback,
 		WALRollbackMinAge: minRootCredRollbackAge,
 		BackendType:       logical.TypeLogical,
+		RotateCredential: func(ctx context.Context, request *logical.Request) error {
+			name, err := b.getDatabaseConfigNameFromRotationID(request.RotationID)
```

Comment:
Can we put this in a method on the databaseBackend type instead of having this whole block within the constructor?

Reply:
Updated